### PR TITLE
Print Config Utility

### PR DESCRIPTION
**Pull request by chases2**
I wrote an experiment to see how well protos would deserialize into json objects (pretty well) and accidentally wrote a useful CLI config printing utility for debugging.

/lint

### CODE REVIEW

File: config/print/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+The config printer is a debugging utility that prints a TestGrid configuration in a
+human-readable format. It will read from the local filesystem or Google Cloud Storage.
+
+## Usage and installation
+
+```sh
+go install ./config/print
+print gs://example/config
+```
+A Bazel `run` command also works, but the utility won't be able to read from the local file system as expected.
```
Comment (fejta) [inline config/print/README.md:12]: https://github.com/kubernetes/test-infra/blob/47b6e0fed7cc39bf87b864302445fe381a9802dc/BUILD.bazel#L10

should address that

**Review by fejta — APPROVED** [review]
/hold

File: config/print/README.md (hunk 1 above)
Comment (chases2) [reply config/print/README.md:12]: Neat! I'll mess with that and see if I can make a quick improvement later.

**Pull request merged**